tests/openwisp2/settings.py

```diff
@@ -106,7 +106,7 @@
 MEDIA_URL = '/media/'
 MEDIA_ROOT = '{0}/media/'.format(BASE_DIR)
 
-PRIVATE_STORAGE_ROOT = '{0}/firmware/'.format(BASE_DIR)
+PRIVATE_STORAGE_ROOT = os.path.join(MEDIA_ROOT, 'private')
```

Comment:
I made this change to make it consistent with docker-openwisp and ansible-openwisp2

https://github.com/openwisp/docker-openwisp/blob/aa6ce9b5af3af672b59e62b3302ff6cd027c3754/images/common/openwisp/settings.py#L247

https://github.com/openwisp/ansible-openwisp2/blob/5a2860e4d37bbe33904628b0f31acc0b1772401e/templates/openwisp2/settings.py#L109